README.md

```diff
@@ -27,19 +27,58 @@ If you have a capable (Nvidia, at least 8GB of VRAM) GPU, it is highly recommend
+This script will train on a small sample dataset called LDC93S1, which can be overfitted on a GPU in a few minutes for demonstration purposes. From here, you can alter any variables with regards to what dataset is used, how many training iterations are run and the default values of the network parameters.
+Feel also free to pass additional (or overriding) `DeepSpeech.py` parameters to these scripts.
+Then, just run the script to train the modified network.
+
+## Checkpointing
+
+During training of a model so called checkpoints will get stored on disk. This takes place at a configurable time interval. Files are also rotated. The purpose of checkpoints is to allow interruption (also in case of some unexpected failure) and later continuation of training without loosing hours of training time. Resuming from checkpoints happens automatically by just (re)starting training with the same `--checkpoint_dir` of the former run.
+
+Be aware however that checkpoints are only valid for the same model geometry they had been generated from. In other words: If there are error messages of certain `Tensors` having incompatible dimensions, this is most likely due to an incompatible model change. One usual way out would be to wipe all checkpoint files in the checkpoint directory or changing it before starting the training.
 
 ## Exporting a model for serving
 
-If the `ds_export_dir` environment variable is set, or the `export_dir` variable is set manually, a model will have been exported to this directory during training. If training has been performed without exporting a model, a model can be exported by setting the variable to the directory you'd like to export to (e.g. `export_dir = os.path.join(checkpoint_dir, 'export')`) and running the model exporting cell manually. If the notebook has been restarted since training, you will need to run all the cells above the training cell first before running the export cell, to declare and initialise the required variables and functions.
+If the `--export_dir` parameter is provided, a model will have been exported to this directory during training.
+Refer to the corresponding [README.md](client/README.md) for information on building and running a client that can use the exported model.
+
+## Distributed computing across more than one machine
+
+DeepSpeech has built-in support for [distributed TensorFlow](https://www.tensorflow.org/deploy/distributed). To get an idea on how this works, you can use the script `bin/run-cluster.sh` for running a cluster with workers just on the local machine.
+
+```bash
+$ bin/run-cluster.sh --help
```

Comment:
To make things consistent you should have the  `DeepSpeech` directory on the command line too
```bash
DeepSpeech$ bin/run-cluster.sh --help
```